packages/flutter_tools/lib/src/device.dart

```diff
@@ -473,6 +474,8 @@ abstract class Device {
   static Future<void> printDevices(List<Device> devices) async {
     await descriptions(devices).forEach(printStatus);
   }
+
+  void dispose() {}
```

Comment:
Nit: add a doc comment on all of the non-override dispose methods

Reply:
done